python/graphstorm/model/gnn.py

```diff
@@ -673,7 +678,14 @@ def get_input_embeds(input_nodes):
                 input_nodes = {data.g.ntypes[0]: input_nodes}
             return {ntype: input_embeds[ntype][ids].to(device) \
                     for ntype, ids in input_nodes.items()}
-        embeddings = model.gnn_encoder.dist_inference(data.g, get_input_embeds,
+        if minibatch:
```

Comment:
we should have a function called `mini_batch_inference` outside `full_graph_inference`

Reply:
It is still full graph inference, but in a mini-batch manner.

Reply:
is there any advantage of computing GNN embeddings of all nodes in a graph?